### DISCUSSION TITLE
Improvements & Projects

### DISCUSSION THREAD

**Discussion by ss-o** [bew/dotfiles#82] [Q&A]
Hey, :wave: 

I came across your dotfiles, and after reviewing them all, would like to propose teamwork if there are some spare minutes in your schedule. Mainly I have a few features/ideas for ZI, however, little support would help to achieve better and faster. If you are interested let me know (sall@w-ss.io). Thanks :octocat: 

Sincerely,

**Comment by bew** [comment]
Hello!

Are you asking if I could do contributions on zi ?
I don't do much dev currently, so I'm not sure I could really help with anything :eyes: 
What were you thinking about?
